input_files/CHP/chp_input_defaults_processing.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import os
 import copy
-import pickle
+import json
```

Comment:
@Bill-Becker do we still need this file now that all the data is in the CHP Tech class?

Reply:
@NLaws No, I can remove it and keep it external to modify defaults as needed.

Reply:
Removed extraneous files in https://github.com/NREL/REopt_Lite_API/pull/78/commits/d1d7250a08b4a1ef51eddd2170e29cdb2f40badf.  Just left a .json with the data used.